src/windows/functions.cpp

```diff
@@ -393,6 +393,12 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveLetterValid(TCHAR driveLetter) {
+  int driveLetterIndex = static_cast<int>(driveLetter) - static_cast<int>('A');
```

Comment:
> I guess one way of avoiding these casts would be to switch the while (logicalDrivesMask) { loop to iterating by integer index instead of currentDriveLetter, and then only converting the integer index to an actual driveLetter at the last possible moment?

Is there any advantage of avoiding the cast?

> Alternatively, maybe it'd be helpful if the MountUtilsLog functions actually specified which currentDriveLetter they're trying to operate on?

I had a branch with these changes, but looks like I forgot to push it :P I'll do so in a minute.

> Something I've just thought of - does this function still behave correctly when there are network shares mounted to drive letters?

I think it should work fine, but I'll test that out.